integration-tests/downscope/downscope_test.go

```diff
@@ -145,11 +145,10 @@ func downscopeQuery(t *testing.T, tt downscopeTest) error {
 	if err != nil {
 		return fmt.Errorf("failed to retrieve object from GCP project with error: %v", err)
 	}
+	defer resp.Body().Close()
```

Comment:
This does not compile. Should be `resp.Body.Close()`

Reply:
My bad, thanks.